playground/header.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<style>
```

Comment:
@SwatiRaoDataEnthusiast We would want to keep the styles and html content separate; to avoid maintainability issues as the file will start to grow moving forward.

- [ ] Move styles into its own file and use <link> tag to include that html